Svg.Editor.Core.Tests/SelectionToolTests.cs

```diff
@@ -104,7 +104,7 @@ public async Task IfUserDrawsSelectionRectangle_AndElementsAreContained_Elements
             Assert.AreEqual(0, Canvas.SelectedElements.Count);
 
             // Act
-            var start = PointF.Create(b1.Left - 1, b1.Top - 1);
+            var start = PointF.Create(b2.Left - 1, b1.Top - 1);
```

Comment:
recreating the two elements it shows this

![grafik](https://github.com/gentledepp/SVG/assets/44953551/7df71d47-65d3-452d-8d36-14fea3799145)

means that selection area was not set correctly.
second elements left was wider :).
Going to refactor other tests and see if they work